CMakeModules/cmake_uninstall.cmake.in

```diff
@@ -0,0 +1,21 @@
+if(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+  message(FATAL_ERROR "Cannot find install manifest: @CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+endif(NOT EXISTS "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt")
+
+file(READ "@CMAKE_CURRENT_BINARY_DIR@/install_manifest.txt" files)
+string(REGEX REPLACE "\n" ";" files "${files}")
+foreach(file ${files})
+  message(STATUS "Uninstalling $ENV{DESTDIR}${file}")
+  if(IS_SYMLINK "$ENV{DESTDIR}${file}" OR EXISTS "$ENV{DESTDIR}${file}")
```

Comment:
[It seems `EXISTS` captures symbolic link file as well](http://www.cmake.org/cmake/help/v3.2/command/if.html?highlight=#command:if). So it's not.

I guess I referred a little bit outdated receipt. [This](http://www.cmake.org/Wiki/RecipeAddUninstallTarget) looks more recent version that uses `execute_process` and doesn't use `IS_SYMLINK`.